packages/fxa-settings/src/styles/ctas.scss

```diff
@@ -0,0 +1,45 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+.cta-neutral {
```

Comment:
@jodyheavener I had this as `cta-default` because I saw it named that in Figma but the actual page showing all the different button types calls it "neutral" and that seems more fitting.